assets/locales/fr.po

```diff
@@ -63,7 +63,7 @@ msgid "Login Forgot password"
 msgstr "Mot de passe oublié ?"
 
 msgid "Authorize Title"
-msgstr "Autoriser %s à accéder à votre profil ?"
+msgstr "Autoriser %s à accéder à votre profil ?"
```

Comment:
This is just a non-breakable space, so the question mark doesn't go to the next line by itself. It wasn't really pretty.